app/policies/comment_policy.rb

```diff
@@ -0,0 +1,20 @@
+class CommentPolicy < ApplicationPolicy
+  attr_reader :user, :comment
+
+  def initialize(user, comment)
+    @user = user
+    @comment = comment
+  end
+
+  def create?
+    true
```

Comment:
I'm not sure allowing create to be true covers all fringe cases, keep a lookout for unwanted permissions!